codegen/src/codegen_rust.rs

```diff
@@ -904,10 +913,10 @@ impl<'model> RustCodeGen<'model> {
         let has_cbor = proto.map(|pv| pv.has_cbor()).unwrap_or(false);
         w.write(
             br#"{
-            async fn dispatch(
-                &self,
-                ctx: &Context,
-                message: &Message<'_> ) -> RpcResult< Message<'_>> {
+            async fn dispatch<'disp__,'ctx__,'msg__>(
```

Comment:
Is there an error or warning that we're trying to prevent by giving each parameter its own lifetime tag?

Reply:
yes unfortunately - I'd prefer to leave them elided, but I couldn't compile "AnySender" without adding lifetimes here. Fortunately this is a method signature that nobody needs to write by hand - it's code-generated.

Reply:
ok cool. I was curious if this would show up in a tooltip hovering over a function